tests/globalkilltest/main_test.go

```diff
@@ -15,13 +15,13 @@
 package globalkilltest
 
 import (
+	"os"
 	"testing"
 
 	"github.com/pingcap/tidb/util/testbridge"
-	"go.uber.org/goleak"
 )
 
 func TestMain(m *testing.M) {
 	testbridge.WorkaroundGoCheckFlags()
-	goleak.VerifyTestMain(m)
+	os.Exit(m.Run())
```

Comment:
So drop the leak test for kill?

Reply:
We don't check it previously and it causes more time to dig out whether to handle these leaks. I don't have such spare time.